packages/components-react/src/index.ts

```diff
@@ -1 +1,3 @@
 export { Button } from './Button';
+export { TopTaskNavigation } from '../../next-templates/src/components/TopTask/TopTaskNavigation';
```

Comment:
Hmm deze exports vind ik gek. Waarom staan die TopTask componenten op zo'n andere plek dan Button?

Reply:
Is alleen voor de exportering van React-componenten (resolved)